dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackerTrackedEntitiesExportControllerTest.java

```diff
@@ -280,6 +281,19 @@ void getTrackedEntityByIdNotFound()
                 .getMessage() );
     }
 
+    @Test
+    void getTrackedEntityCsvFormat()
+    {
+        WebClient.HttpResponse response = GET( "/tracker/trackedEntities.csv?program={programId}&orgUnit={orgUnitId}",
+            program.getUid(), orgUnit.getUid() );
+
+        assertEquals( HttpStatus.OK, response.status() );
+
+        assertTrue( response.header( "content-type" ).contains( "application/csv" ) );
+        assertTrue( response.header( "content-disposition" ).contains( "filename=\"trackedEntities.csv\"" ) );
+        assertTrue( response.content().toString().contains( "trackedEntity,trackedEntityType" ) );
+    }
```

Comment:
I think we agreed to use `assertAll` for multipe assertions?

Reply:
I was not aware of that, but I'll change this to use `assertAll` instead.